src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -140,4 +140,22 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses {@code String args} and extracts out the type of Person (Tutor/Student).
+     *
+     * @return Returns the string at the index of args.
+     */
+    public static PersonType parsePersonType(String args) throws ParseException {
+        // Allows for command to be valid even with multiple whitespaces within the command.
+        // For e.g. "add    t   n/..." will be a valid command read as "add t n/...".
+        String formattedString = args.replaceAll("\\s{2,}", " ").trim();
+        String[] parsedString = formattedString.split(" ");
+        String personType = (String) Array.get(parsedString, 1);
+        try (PersonType.detectPersonType(personType)) {
```

Comment:
Not sure if it's incomplete or what but I don't think try takes in an argument? 😫